src/natpmp.cpp

```diff
@@ -62,7 +62,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 namespace libtorrent {
 
-struct TORRENT_EXPORT pcp_error_category : boost::system::error_category
```

Comment:
this raises an interesting question (although, independent of your patch). Are the concrete error category classes implementation details, not available to clients?

Reply:
I suppose none of them are defined in a header

Reply:
I had a similar question when I realized they are confined to the `.cpp`, but... I don't know